src/core/cp-calculator/cp-calculator.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { expect } from 'chai';
+import { CPCalculator } from './cp-calculator';
+
+describe('Class: CPCalculator', () => {
+    it('should output the correct values', () => {
+        // Source: http://www.imore.com/pokemon-go-gen-2
+        let cp: number;
+        // Bayleef
+        cp = CPCalculator.Calculate(122, 155, 120);
+        expect(cp).to.equal(1296);
+
+    });
+});
```

Comment:
Can we add a test case for another pokemon or two? Here's one for Meganium:

it('should output the correct values for Meganium', () => {
        // source: https://rankedboost.com/pokemon-go/generation-2-tier-list/
        let cp: number;
        cp = CPCalculator.Calculate(160, 168, 202);
        expect(cp).to.equal(2227);
    });